_data/collection-index.yml

```
@@ -308,3 +308,8 @@
   contact: https://github.com/heckj/devcontainer-swift-additions/issues
   repository: https://github.com/heckj/devcontainer-swift-additions
   ociReference: ghcr.io/heckj/devcontainer-swift-additions
+- name: kubeseal cli feature
+  maintainer: Andrii Tararaka
+  contact: https://github.com/gickis/devcontainer-features/issues
+  repository: https://github.com/gickis/devcontainer-features
+  ociReference: ghcr.io/gickis/devcontainer-features/kubeseal
```

```suggestion
  ociReference: ghcr.io/gickis/devcontainer-features
```

This index adds an entry of the `ociReference` for [collection metadata](https://containers.dev/implementors/features-distribution/#devcontainer-collection-json) instead of individual Features. Thus, the author needs to only add a single entry irrespective of the number of Features that might exist in the repo.

Can you update this? thanks!

---

Updated according to suggestions